BUILD_MESON

```diff
@@ -122,20 +129,37 @@ Enable "unit-tests" option to run the test suite:
      
 Unlike Dinit's Make build system, We don't enable -fsanitize by default so if you need that use this option in meson setup step:
```

Comment:
This is not correct, Dinit's make-based build system enables -fsanitize only for tests for by default.